addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -0,0 +1,208 @@
+/** @odoo-module **/
+
+import options from 'web_editor.snippets.options';
+import {websiteDomain} from 'website.utils';
+
+options.registry.SocialMedia = options.Class.extend({
+    /**
+     * @override
+     */
+    async willStart() {
+        this._super(...arguments);
+        const websiteId = websiteDomain(this)[2][2];
+        const target = this.$target[0];
+        // Fetch URLs for db links.
+        const res = await this._rpc({
+            model: 'website',
+            method: 'search_read',
+            args: [[['id', '=', websiteId]], ['social_facebook', 'social_twitter', 'social_youtube',
+                'social_instagram', 'social_linkedin', 'social_github']],
+            limit: 1,
```

Comment:
search_read by id is just a read 
```suggestion
            method: 'read',
            args: [websiteId, ['social_facebook', 'social_twitter', 'social_youtube',
                'social_instagram', 'social_linkedin', 'social_github']],
```